debs/siegfried/debian/rules

```diff
@@ -2,7 +2,7 @@
 
 export DH_OPTIONS
 
-export DH_GOPKG := github.com/richardlehane/siegfried
+export DH_GOPKG := github.com/richardlehane/siegfried/cmd...
```

Comment:
From a go perspective the binary would be built from the `cmd` dir here I believe, but what is the reason behind this change in the package this time around? (Curiosity more than anything).

Reply:
https://github.com/artefactual-labs/am-packbuild/pull/251#issuecomment-578112922

Reply:
To be clear I don't feel this is an issue, I was just wondering technically why so that I can learn, but that commend described why we updated `Go` not why we changed this path, unless I am missing something?

Reply:
I think that `DH_GOPKG := github.com/richardlehane/siegfried` was only working because of the way that golang.pm was building the target for `go install`. That probably stopped working when Richard adopted modules in Siegfried.

@mamedin, I think that golang.pm would prefer something like `github.com/richardlehane/siegfried/cmd` instead. The discovery process would still work, e.g the build command results the same: `go install -v -tags archivematica github.com/richardlehane/siegfried/cmd/roy github.com/richardlehane/siegfried/cmd/sf`. But `dh_auto_install` names the temporary directory as `cmd` instead of `cmd...`.

Reply:
@ross-spencer FYI golang.pm is the thing that Debian uses to build Go packages. We have our own copy here in this repo (https://github.com/artefactual-labs/am-packbuild/blob/stable/1.10.x/debs/siegfried/files/golang.pm) because we need to build with `-tags=archivematica` (https://github.com/richardlehane/siegfried/blob/master/pkg/config/archivematica.go).